### PR TITLE
Strict mode activated [DRAFT - WIP]

### PR DESCRIPTION
**Pull request by arivepr**
Unsure of how to fix the last error popping up for me:

6E905984-514D-4A9B-9206-9FE8C8E64375_1_105_c

### CODE REVIEW

File: src/browser/index.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ function getChromiumExectuablePath() {
   }
 }
 
-const CHROMIUM_PATH = config.IS_PRODUCTION && getChromiumExectuablePath();
+const CHROMIUM_PATH = getChromiumExectuablePath();
```
Comment (Hyperkid123) [inline src/browser/index.ts:31]: the production condition is critical. Are we sure we can remove it?

File: src/templates/automation-analytics/Components/ReportComponents/Standard/Components/TemplateExplorerExpandedRow/index.tsx [hunk 2]
```diff
@@ -3,6 +3,7 @@ import { Flex, FlexItem, Grid, GridItem } from '@patternfly/react-core';
 import { ExpandableRowContent, Td, Tr } from '@patternfly/react-table';
 
 import Breakdown from './BreakdownChart';
+// import CategoryCount from './BreakdownChart';
```
Comment (Hyperkid123) [inline src/templates/automation-analytics/Components/ReportComponents/Standard/Components/TemplateExplorerExpandedRow/index.tsx:6]: Why we can't use this value?

Comment (arivepr) [reply src/templates/automation-analytics/Components/ReportComponents/Standard/Components/TemplateExplorerExpandedRow/index.tsx:6]: we are! it was being defined where it wasn't used, just moved it to this file instead :) 

I'll clean it up

File: src/browser/index.ts (hunk 1 above)
Comment (Hyperkid123) [reply src/browser/index.ts:31]: It should have falsy value

**Review by Hyperkid123 — APPROVED** [review]
LGTM

**Pull request merged**